packages/access-api/src/models/delegations.js

```diff
@@ -116,14 +117,18 @@ class UnexpectedDelegation extends Error {
 }
 
 /**
- * @param {Pick<DelegationRow, 'bytes'>} row
+ * @param {Pick<import('../types/access-api-d1').DelegationsV2Table, 'bytes'>} row
  * @returns {Ucanto.Delegation}
  */
 function rowToDelegation(row) {
   /** @type {Ucanto.Delegation[]} */
   let delegations = []
+  // kysely/sqlite/d1/various-miniflare-versions sometimes only give an Array here.
+  // (GenericPlugin tries to ensure Uint8Array, but it can't reliably detect when to cast Array -> Uint8Array)
```

Comment:
When I upgraded miniflare, it changed the way it was returning sqlite bytes from before. Before, the `GenericPlugin` hugo wrote would get the bytes back in a way that `isBuffer` would succeed. Now it doesn't (which is actually nice, because that isBuffer check didn't work on cloduflare anyway).

I should try upgrading kysely-d1 to latest to see how that affects things. https://github.com/aidenwallis/kysely-d1/pull/19

Reply:
done in https://github.com/web3-storage/w3protocol/pull/598